src/views/FacilityDetails.vue

```diff
@@ -1113,9 +1115,7 @@ ion-card-header {
 }
 
 
-/*Height of segment is defined now since their are less list items. Will remove it later */
 .segments {
```

Comment:
do we need to use a class or can we just select the element? @amansinghbais

Reply:
@dt2patel sir,  div with segments class contains 3 child div inside it and some css is applied to those div tag with the help of 'segments' class as ".segments div". So if we don' t want to use segments class, we need to give some identifier(class) to the div inside for adding css on them.